lib/checks/forms/submit-button.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "submit-button",
+  "evaluate": "submit-button.js",
+  "metadata": {
+    "impact": "serious",
+    "messages": {
+      "pass": "Form has a way to submit content",
+      "fail": "Form can not be submited by any indicated element"
```

Comment:
We'll need an "incomplete" message here telling the user to check that this form doesn't get automatically submitted. You can remove the "fail" key as that wouldn't be used anymore.